editorconfig-maven-plugin/src/main/java/org/ec4j/maven/AbstractEditorconfigMojo.java

```diff
@@ -280,6 +292,23 @@ public void execute() throws MojoExecutionException, MojoFailureException {
         if (excludeNonSourceFiles) {
             excls.addAll(Constants.DEFAULT_EXCLUDES);
         }
+
+        if (excludesFile != null) {
+            log.debug("Using excludesFile '{}'", excludesFile);
+            try {
+                try (BufferedReader reader = Files.newBufferedReader(excludesFile.toPath(), charset)) {
+                    String line;
+                    while ((line = reader.readLine()) != null) {
+                        if (!line.isEmpty() && !line.startsWith("#")) {
+                            excls.add(line);
+                        }
+                    }
+                }
+            } catch (IOException e) {
+                throw new MojoExecutionException("Error reading excludesFile '" + excludesFile + "'", e);
```

Comment:
```suggestion
                throw new RuntimeException("Error reading excludesFile '" + excludesFile + "'", e);
```

and please remove the throws declarations in the method and JavaDoc.

Reply:
Done.